modules/gapi/src/backends/fluid/gfluidcore_simd_sse41.hpp

```diff
@@ -554,7 +554,162 @@ CV_ALWAYS_INLINE void calcRowLinear_8UC_Impl_<3>(uint8_t* dst[],
         }
     }
 }
-} // namespace sse42
+
+CV_ALWAYS_INLINE void v_gather_pairs(const float src[], const int* mapsx,
+                                     v_float32x4& low, v_float32x4& high)
+{
+    __m128 l = _mm_setzero_ps();
+    l = _mm_loadl_pi(l, (const __m64*)&src[mapsx[0]]);  // pair of floats
+    l = _mm_loadh_pi(l, (const __m64*)&src[mapsx[1]]);
+    low.val = l;
```

Comment:
is it possible to load _mm_loadl_pi/_mm_loadh_pi directly into `low.val`  and 'high.val'?